packages/storybook-docs/src/globalStyles.scss

```diff
@@ -0,0 +1,373 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
```

Comment:
what is this for ? It is overridden for the body just under

Reply:
As discussed, this set the generic value of all fonts, and it becomes the basis for 1rem. 1re = 64.5% of (usually) 16px. 
This trick makes it so 1rem = 10px. 
On the body, you build ont top of that rule by setting a default font size.